### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by Bakkerwj** [biemond/solaredge.modbus#2]
Graag de status weer in de Flow's..

![image](https://user-images.githubusercontent.com/49026657/158626865-b64b7bd4-f0b9-4485-aa7b-32dcd06ed079.png)

**Comment by biemond** [comment]
perfect, gaan we doen

**Comment by biemond** [comment]
ben nu aan testen en wacht totdat het donker wordt voor een paar status veranderingen.  

ik stuur je later een link

**Comment by Bakkerwj** [comment]
Super! Thanx..

Met vriendelijke groet,

Willem Jan Bakker

**Comment by biemond** [comment]
kan je deze versie testen https://homey.app/nl-nl/app/solaredge.modbus/SolarEdge/test/

**Comment by Bakkerwj** [comment]
Gedaan, maar kan morgen pas zien of t goed werkt. 

Met vriendelijke groet,

Willem Jan Bakker

**Comment by biemond** [comment]
yep

**Comment by biemond** [comment]
nog steeds producing voor mij

**Comment by Bakkerwj** [comment]
Hij gaf vanmorgen keurig de melding Producing…

Alleen als ik NU een status opvraag geeft ie FAULT, hij geeft alleen de juiste status op t moment van wijzigen maar niet meer “tussendoor” als je daar om vraagt.



Met vriendelijke groet,

Willem Jan Bakker

**Comment by biemond** [comment]
inderdaad,  dat is een trigger alleen als er iets wijzigt. anders wordt je gek en krijg je een trigger elke 20 seconden.  

maar wil je een ook condtie , zoiets   de zon schijnt en status = producing dan doe dit.

**Comment by Bakkerwj** [comment]
“voorheen” kon ik met de flow iid bij een wijziging een status wijziging krijgen, maar als ik de “status” opvroeg kreeg ik m ook, dat wil nu niet meer.

Dus idd bijv, 10:00 uur status is producing, doe de zonneschermen naar beneden. oid..

Met vriendelijke groet,

Willem Jan Bakker
Boerestreek 1
8757JR Gaast
Mob. +31653217707

**Comment by biemond** [comment]
mooi, dat ga ik toevoegen.

**Comment by biemond** [comment]
je kan nu wel tijdelijk invertor is higher gebruiken , bijvoorbeeld meer dan 500

**Comment by Bakkerwj** [comment]
Ja, Ik gebruik nu  ook “het vermogen is veranderd” EN "Solar+Batt power is groter dan 2500” 

En dergelijke flows, onder meer om met onze warmtepomp warmte te bufferen als er veel zon is..



Met vriendelijke groet,

Willem Jan Bakker
Boerestreek 1
8757JR Gaast
Mob. +31653217707

**Comment by biemond** [comment]
nice

**Comment by biemond** [comment]
kun je deze versie testen https://homey.app/a/solaredge.modbus/test/
<img width="790" alt="Screen Shot 2022-03-18 at 15 14 13" src="https://user-images.githubusercontent.com/1085004/159019112-ef2b6385-f987-414a-9fb3-3d497de888a5.png">

**Comment by Bakkerwj** [comment]
De inverteren hebben een eigen id ja, ik heb daar eind vorig jaar met Pieter contact over gehad, omdat het eerst niet werkte met 2 inverter’s.
Hij heeft toen e.e.a. aangepast en toen werkte het, met elke hun eigen ID. Sinds een update van jou werkte inverter 2 ineens niet meer, toen wat aan het uitproberen geweest en nadat ik 2 op 1 zetten werkten ze beide. Prima natuurlijk maar t is wat vreemd.





Met vriendelijke groet,

Willem Jan Bakker
Boerestreek 1
8757JR Gaast
Mob. +31653217707

**Comment by biemond** [comment]
bedankt , Ik ga even kijken.

kun je trouwens op beide  invertor met batterij als device gebruiken. dan zou je flow action op beide kunnen gebruiken. misschien helpt dan de 2de invertor mee met opladen.

**Comment by biemond** [comment]
kan je diagnostic report maken met verschillende unit ids , ben benieuwd wat ik dan zie.  Andere hebben het werkend met verschillende ids

**Comment by Bakkerwj** [comment]
’t is hier donker….

Met vriendelijke groet,

Willem Jan Bakker

**Comment by Bakkerwj** [comment]
Hierbij. 

1492b82d-664a-4bae-bb7f-2688c4a9e822

Met vriendelijke groet,

Willem Jan Bakker
Boerestreek 1
8757JR Gaast
Mob. +31653217707
